src/lib/capiMultiple.ts

```diff
@@ -13,21 +13,50 @@ function addHeadlineKicker(
 			// eslint-disable-next-line @typescript-eslint/no-non-null-assertion -- headline should always be defined
 			cardData[i]!.articleHeadline = overrideCards[i]?.headline as string;
 			// eslint-disable-next-line @typescript-eslint/no-non-null-assertion -- we define the kicker here - it's an optional parameter
-			cardData[i]!.kicker = overrideCards[i]?.kicker as string;
+			cardData[i]!.articleKicker = overrideCards[i]?.kicker as string;
 		}
 	}
 
 	return cardData;
 }
 
-function retrieveCapiData(cards: CapiCardOverride[], seriesUrl: GAMVariable) {
-	let request = `${apiEndpoint}?k=${encodeURI(seriesUrl)}`;
-	cards.forEach((card) => {
-		if (card.url) {
-			request += `&t=${encodeURI(card.url)}`;
+function addOverridesToCardData(
```

Comment:
can this replace the above `addHeadlineToKicker` function? Seems to be doing something very similar

Reply:
The overwriting rules are slightly different so I couldn't replace it. I've refactored both functions to hopefully make it easier to see what's going on. Let me know if you have any thoughts.